src/test/java/UserSmokeTests.java

```diff
@@ -94,4 +119,14 @@ private void verifyUser(Response response, int userId, String username, String f
                 .body("phone", equalTo(phone))
                 .body("userStatus", equalTo(userStatus));
     }
+
+    private boolean userExists(String username) {
```

Comment:
Move such methods from private here to a separate class like UserUtils or UserSteps